action.yml

```
@@ -13,8 +13,11 @@ inputs:
 
       If this is not given, the action will try and install the version specified in the `rust-toolchain` file.
     required: false
-  target:
-    description: Target triple to install for this toolchain
+  targets:
+    description: Comma-separated list of additional target triples to install for this toolchain
+    required: false
+  targets:
```

```suggestion
  target:
```
It's expected that the alias to be named `targets` (with the `s`) ?

---

Good catch, thanks!